clang/test/SemaSYCL/kernel-function-type.cpp

```diff
@@ -0,0 +1,32 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -ast-dump %s | FileCheck %s
```

Comment:
I would expect `-verify` flag.

Reply:
I agree, this shouldn't check the AST, it should just check the static-assert.

Reply:
Should I remove the -ast-dump flag?

Reply:
I think if you shouldn't check the AST, then you don't need the `-ast-dump` flag.